<!-- Make sure launch nav_preapare first -->
<launch>
    <param name="/use_sim_time" value="true" />

    <include file="$(find thewalkingdead)/launch/nav_prepare.launch" />
    
    <node type="task9.py" name="task9" pkg="thewalkingdead" output="screen"/>
</launch>